src/assessments/links/test-steps/link-purpose.tsx

```diff
@@ -77,11 +77,6 @@ const propertyBagConfig: PropertyBagColumnRendererConfig<LinkPurposePropertyBag>
         displayName: 'Accessible name',
         defaultValue: NoValue,
     },
-    {
-        propertyName: 'accessibleDescription',
```

Comment:
This is just a curiosity question, how come this gets removed from link-purpose?

Reply:
ah this is a mistake! reverting this! thanks!